.github/workflows/test.yml

```diff
@@ -17,7 +35,7 @@ jobs:
         node-version: '14.x'
     - name: Install dependencies
       run: npm install --dev
-    - name: Lint
+    - name: Lint # TODO: set linter to be scoped to not modules
```

Comment:
Is this TODO still needed? I see the ignore pattern was added in the `.eslintrc`

Reply:
good catch! removed the comment tyty